### PR TITLE
Add release notes: zbus

### PR DESCRIPTION
**Pull request by rodrigopex**
Add zbus information.

### CODE REVIEW

**Review by stephanosio — APPROVED** [review]

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -1338,6 +1343,55 @@ Libraries / Subsystems
   * Added the linear range API to map values in a linear range to a range index
     :zephyr_file:`include/zephyr/sys/linear_range.h`.
 
+* Zbus
+
+  * Added the :ref:`zbus` to Zephyr.
+
+    - Channel-centric multi-paradigm (message-passing and publish-subscribe) communication message bus;
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:1350]:
```suggestion
    * Channel-centric multi-paradigm (message-passing and publish-subscribe) communication message bus;
```

File: doc/releases/release-notes-3.3.rst [hunk 2]
```diff
@@ -1338,6 +1343,55 @@ Libraries / Subsystems
...
+    - Virtual Distributed Event Dispatcher;
+    - Observers can be listeners (synchronous) and subscribers (asynchronous);
+    - One-to-one, one-to-many, and many-to-many communications;
+    - Persistent messages distributed by shared-memory approach;
+    - Delivery guarantee only for listeners;
+    - Uses mutex to control channels access;
+    - Added the following samples:
+
+      - :ref:`zbus-hello-world-sample`
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:1359]:
```suggestion
      * :ref:`zbus-hello-world-sample`
```

**Review by stephanosio — CHANGES_REQUESTED** [review]
Looks good overall. Just a minor comment about the bullet point usage.

File: doc/releases/release-notes-3.3.rst (hunk 1 above)
Comment (rodrigopex) [reply doc/releases/release-notes-3.3.rst:1350]: I will fix that.

File: doc/releases/release-notes-3.3.rst (hunk 2 above)
Comment (rodrigopex) [reply doc/releases/release-notes-3.3.rst:1359]: I will fix that.

File: doc/releases/release-notes-3.3.rst (hunk 1 above)
Comment (rodrigopex) [reply doc/releases/release-notes-3.3.rst]: Done

File: doc/releases/release-notes-3.3.rst (hunk 2 above)
Comment (rodrigopex) [reply doc/releases/release-notes-3.3.rst]: Done

**Review by rodrigopex — COMMENTED** [review]
Requests attended.

**Review by stephanosio — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Pull request merged**